### DISCUSSION TITLE
sankey confusion with Source/Target label

### DISCUSSION THREAD

**Discussion by tomresing** [apache/superset#27890] [Ideas]
## Screenshot
![SanKeySourceTargetSupersetIssue](https://github.com/apache/superset/assets/2660081/12b9a1e2-cc2f-4bfc-817f-ef4e2ce946e1)

## Description

The label "Source/Target" and the tooltip for it could be improved to reduce confusion. With the current label, it's not clear which selection applies to which part of the chart and the tooltip doesn't help.

I'd suggest that "SOURCE AND TARGET" would be a better label and "The first column you select will be the source of the flow and the second will be the target."

## Design input
Review the suggestion and provide any alternative

**Comment by rusackas** [comment]
Moving this to a Discussion, since it's not really a _bug_ per se, but... I agree with you. 

Looking at an example, it's easier to see how to reason about this:
<img width="1379" alt="image" src="https://github.com/apache/superset/assets/812905/b909b528-eb52-4cc8-a0df-07726b65713b">

That said, it's unclear when you look at the controls with a blank slate. We're very open to suggestion, or PRs, on this. In my mind, it might make sense to have TWO controls:
* Source Column (tooltip: "the column in your dataset that will be matched/connected to your Target Column to display the row's value")
* Target Column (tooltip: "the column in your dataset that will be matched/connected to your Source Column to display") the row's value")

Something kinda like that anyway... curious if others have thoughts here (cc @kasiazjc )

**Reply by tomresing** [reply]
Yes. Two controls sounds good to me, too. 
I'd suggest the shortest, simplest language necessary would be best.
The label for the first could be Source and the second could be target. I'm not sure if you need an info bubble unless there's something you need to add. The tooltip suggestions you've made don't make things more clear for me.

Just to be clear, what happens if you add a third column in the current experience? I don't think I tried that and I can't picture how that would work.
